Documentation/ResultsStructure.md

```diff
@@ -0,0 +1,150 @@
+| GRF_r | ground reaction forces right side | mesh x 3 | [N] |
+| GRF_l | ground reaction forces left side | mesh x 3 | [N] |
+| GRM_r | ground reaction moments right side | mesh x 3 | [Nm] |
+| GRM_l | ground reaction moments left side | mesh x 3 | [Nm] |
+| COP_r | centre of pressure right side (in ground frame) | mesh x 3 | [m] |
+| COP_l | centre of pressure left side (in ground frame) | mesh x 3 | [m] |
+| idx_stance_r | indices of meshpoints with ground contact on right side | <mesh x 1 | [-] |
+| idx_stance_l | indices of meshpoints with ground contact on left side | <mesh x 1 | [-] |
+
+### R.misc
+| Field | Description | Dimension | Unit |
+|------ | ----------- | ---- | ---- |
+| body_mass | body mass | 1 x 1 | [kg] |
+| body_weight | body weight | 1 x 1 | [N] |
+
+### R.kinetics
+| Field | Description | Dimension | Unit |
+|------ | ----------- | ---- | ---- |
+| T_ID | inverse dynamics (inertial and contact) | mesh x coordinates | [Nm or N] |
+| T_spring | coordinate moments by springs | mesh x coordinates | [Nm or N] |
+| T_damping | coordinate moments by dampers | mesh x coordinates | [Nm or N] |
+| T_limit | coordinate moments by limit torques | mesh x coordinates | [Nm or N] |
+
+### R.ligaments
+| Field | Description | Dimension | Unit |
+|------ | ----------- | ---- | ---- |
+| l | ligament length | mesh x ligaments | [m] |
+| v | ligament lengthening velocity | mesh x ligaments | [m/s] |
+| F | ligament force | mesh x ligaments | [N] |
+| strain | ligament strain | mesh x ligaments | [%] |
+| stress | ligament stress | mesh x ligaments | [MPa] |
+| moment | coordinate moment (or force) due to ligaments | mesh x coordinates | [Nm or N] |
+| moment | coordinate power due to ligaments | mesh x coordinates | [W] |
+
+### R.spatiotemp
+| Field | Description | Dimension | Unit |
+|------ | ----------- | ---- | ---- |
+| step_length_r | step length of right leg | 1 x 1 | [m] |
+| step_length_l | step length of left leg | 1 x 1 | [m] |
+| stance_r | duration of stance phase of right leg, relative to gait cycle | 1 x 1 | [%] |
+| swing_r | duration of swing phase of right leg, relative to gait cycle | 1 x 1 | [%] |
+| stance_l | duration of stance phase of left leg, relative to gait cycle | 1 x 1 | [%] |
+| swing_l | duration of swing phase of left leg, relative to gait cycle | 1 x 1 | [%] |
+| double_support | duration of double support phase, relative to gait cycle | 1 x 1 | [%] |
+| stride_freq | stride frequency | 1 x 1 | [Hz] |
+| step_width_COP | step width, calculated based on centre of pressure | 1 x 1 | [m] |
+| dist_trav | distance traveled, calculated based on floating base forward position | 1 x 1 | [m] |
+
+### R.metabolics
+
+#### R.metabolics.Bhargava2004
+
+| Field | Description | Dimension | Unit |
+|------ | ----------- | ---- | ---- |
+| Edot_gait | Metabolic energy rate of each muscle. [$\dot{E}$ in equation 1](https://doi.org/10.1016/S0021-9290(03)00239-2) | mesh x muscles | [W/kg] |
```

Comment:
currently it shows like this
![image](https://github.com/user-attachments/assets/59f9e8e2-4ba4-488c-beab-1f7c3a90c161)

```suggestion
| Edot_gait | Metabolic energy rate of each muscle. $\dot{E}$ [in equation 1](https://doi.org/10.1016/S0021-9290(03)00239-2) | mesh x muscles | [W/kg] |
```